pages/research_software_stories/xah_research_software_story.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Research Software Story - xAODAnaHelpers"
+description: "Analysis Framework for High-Level ATLAS Data"
+contributors: ["Michael Sparks", "Caterina Doglioni", "Tobias Fitschen"]
+page_id: xah_research_software_story
+type: research_software_story
+---
+
+## The Problem
+
+Experiments at the Large Hadron Collider (LHC), like ATLAS, generate vast
+amounts of processed collision data in a format called xAOD.  Physicists
+must analyse this data for physics signals using calibrations, object
+selections, and corrections.  Historically, each analyst largely wrote their
+own code, leading to duplicate work, inconsistent methods, and
+reproducibility challenges.  The community needed a centralised analysis
+framework to standardise workflows and ease maintenance.
+
+## The Community
+
+xAODAnaHelpers (xAH) is a common analysis framework widely used within the
+ATLAS experiment.  It was created in 2015 by Giordon Stark and passed to new
+maintainers (Tobias Fitschen and Michael Hank) in 2023.
+
+The xAH community consists mostly of ATLAS physicists (often early-career
+PhD students).  They have strong physics expertise but less software
+engineering experience.  Most contributions are user-driven via GitHub merge
+requests, and communication happens on a mailing list and through GitHub
+issues.
+
+## Technical Aspects
+
+xAH is a modular framework written mainly in C++, with Python 3 for
+configuration and job steering.  It applies calibrations, object selections,
+and corrections to ATLAS xAOD data, producing structured outputs (ntuples
+and histograms).  The framework integrates directly with ATLAS's official
+software Athena to ensure consistent configuration.
+
+Each analysis is configured with one Python file defining the
+sequence of algorithms, which enhances reproducibility.  Jobs are portable
```

Comment:
This is an interesting point - having one Python file defining the sequence of algorithms - you mentioned that this enhances reproducibility - it would be good to add a reason e.g. ', which enhances reproducibility because ...'